### PR TITLE
Endpoints/article

### PR DESCRIPTION
**Pull request by hypeofpipe**
#### Why?
Add Articles support

#### How?
- Add tests
- Add functionality
- Add documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,76 @@ client.useRequestOpts({
 
 ## Examples
 
+### Articles
```
Comment (johnnynader) [inline README.md:80]: could you move articles after admin, to preserve alphabetical order

File: lib/admin/admin.types.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+    type: 'admin';
+    id: string;
+    name: string;
+    email: string;
+    job_title: string;
+    away_mode_enabled: boolean;
+    away_mode_reassign: boolean;
+    has_inbox_seat: boolean;
+    team_ids: Array<number>;
+    avatar: string;
```
Comment (johnnynader) [inline lib/admin/admin.types.ts:11]: why this change?

File: lib/article/article.types.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    updated_at: Timestamp;
+    url: string;
+    parent_id: string;
+    parent_type: string;
+    default_locale: string;
+    translated_content: TranslatedContentObject;
+    statistics: StatisticsObject;
+};
+
+export enum Locales {
```
Comment (johnnynader) [inline lib/article/article.types.ts:22]: doesn't typescript have a type for this? seems like something that should be there

File: lib/client.ts [hunk 4]
```diff
@@ -97,7 +105,7 @@ export default class Client {
     }
     initiateAxiosInstance(): Axios {
         const defaultHeaders = {
-            'User-Agent': 'intercom-node-client/3.0.0',
+            'User-Agent': `intercom-node-client/${packageJson.version}`,
```
Comment (johnnynader) [inline lib/client.ts:108]: Nice 👌

File: lib/admin/admin.types.ts (hunk 2 above)
Comment (hypeofpipe) [reply lib/admin/admin.types.ts:11]: Seems like linter was applied for this file for the first time :)

File: lib/article/article.types.ts (hunk 3 above)
Comment (hypeofpipe) [reply lib/article/article.types.ts:22]: AFAIK, no :)

File: lib/client.ts (hunk 4 above)
Comment (hypeofpipe) [reply lib/client.ts:108]: Alt Text

**Review by johnnynader — APPROVED** [review]
